source/_integrations/iotmeter.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: IoTMeter
+description: Instructions on how to integrate IoTMeter within Home Assistant.
+ha_category:
+  - Energy
+ha_release: 2024.6
+ha_iot_class: Cloud Polling
+ha_domain: iotmeter
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@lipic'
+ha_integration_type: integration
+---
+
+The **IoTMeter** {% term integration %} allows you to monitor and manage your [IoT meters](http://evracing.cz/iotmeter/iotmeter_EN.pdf) within Home Assistant. This integration provides real-time data and control over your connected devices.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The {% term integration %} adds the following sensors:
+
+- Total power (unit: kW)
```

Comment:
```suggestion
- **Total power** (unit: kW):
```
We use bold to mark UI strings. I also added a colon. Otherwise, It would look like this:
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/27e85345-e2a4-42b5-9e7b-c795a21accba)